lib/event_tracer/datadog_logger.rb

```diff
@@ -18,15 +18,17 @@ class DatadogLogger < BasicDecorator
       gauge: :gauge,
       set: :set,
       histogram: :histogram
-    }
+    }.freeze
     DEFAULT_METRIC_TYPE = :count
     DEFAULT_COUNTER = 1
+    DEFAULT_TAGS = [:environment].freeze # These should be appended by the Logger itself
 
-    attr_reader :allowed_tags
+    attr_reader :allowed_tags, :default_tags
 
     def initialize(decoratee, allowed_tags: [])
```

Comment:
in the example code in the jira ticket there was a default_tags arg here, however i didnt really understand why. if the tags are considered to be default, why should they be externally modifiable? 🤔

Reply:
Okay, certainly doable, however this means this gem + all the applications have to be modified. Originally we did in the shared gem, because that meant that only that gem needed to be modified and it was quite convenient. However I guess it will be cleaner for the long term this way.

Reply:
> Okay, certainly doable, however this means that this gem + all the applications have to be modified

Only the application that needs default tags will need to be modified. For example, I don't think NN really care about `uat` environment.

On the other hand, the current implementation will require modification when other applications have other requirements. For example, if GH wants to add `app_name` as default tag, we'll need to modify the gem, and it will affect LE which needs only `environment` as the default tags.